clippy_lints/src/literal_representation.rs

```diff
@@ -22,11 +22,16 @@ declare_clippy_lint! {
     ///
     /// ### Example
     /// ```rust
-    /// // Bad
-    /// let x: u64 = 61864918973511;
+    /// # let _: u64 =
+    /// 61864918973511
+    /// # ;
```

Comment:
For these ones, I thought it looked better to just show the literals themselves. Since it applies in all contexts I didn't think the `let` statement was all that important for the reader to see.

Reply:
Alright, I'm a bit torn, since I think it looks weird in the code, but the main focus should be the lint list documentation, where this will show correctly.